lib/diagnostic.js

```diff
@@ -41,7 +69,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.deleteOne({motto:"As High as Honor"});
```

Comment:
This will delete the entire document.  Use `update` with `$unset` instead.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```